src/modules/Average/Module.js

```diff
@@ -42,8 +42,8 @@ module.exports = function Average(options, UI) {
       // report back and store average in metadata:
       options.step.metadata.averages = sum;
 
-      // TODO: refactor into a new "display()" method as per https://github.com/publiclab/image-sequencer/issues/242
-      if (options.step.inBrowser && options.step.ui) $(options.step.ui).find('.details').append('<p><b>Averages</b> (r, g, b, a): ' + sum.join(', ') + '</p>');
+      if(options.step.average === undefined) options.step.average = '';
```

Comment:
Can you add a space after the `if` here? Not a big deal but helps us keep code tidy over the long term! Thanks!